pkg/sanity/controller.go

```diff
@@ -443,6 +445,56 @@ var _ = DescribeSanity("Controller Service", func(sc *SanityContext) {
 			Expect(err).NotTo(HaveOccurred())
 			cl.UnregisterVolume(name)
 		})
+
+		It("should not fail when creating volume with maximum-length name", func() {
```

Comment:
I think this is debatable. The spec makes a guarantee to the CSI driver that the name length won't exceed 128 bytes. It's then reasonable to assume that CSI drivers support names up to that size. Otherwise how can a CO know how long it can make the names without breaking some CSI driver which happens to support only shorter names? Therefore I think this test is okay.

Reply:
In a driver implementation it is ok, but in csi-sanity we cannot assume any situation. This PR is making a clear decision that it wants the driver to support 128 chars. That is not in the spec.

Reply:
@pohly Let me know that I was talking about the ID. My mistake. This is a good test.